capplets/appearance/appearance-main.c

```diff
@@ -56,6 +57,12 @@ init_appearance_data (int *argc, char ***argv, GOptionContext *context)
 
   data->filechooser_settings = g_settings_new (FILECHOOSER_SCHEMA);
   data->interface_settings = g_settings_new (INTERFACE_SCHEMA);
+
+  if (GDK_IS_X11_DISPLAY ((gdk_display_get_default())))
+    data->interface_gnome_settings = g_settings_new (INTERFACE_GNOME_SCHEMA);
+  else
+    data->interface_gnome_settings = NULL;
```

Comment:
This looks like it is the wrong way around, isn't it?

Reply:
Probably got confused by seeing it shown here reversed

Reply:
Fixed this in the latest force-push